### PR TITLE
add customer tag

### PR DESCRIPTION
**Pull request by yikyak02**
add customer red coloured customer tag for easy identification

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddCustomerCommand.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
  */
 public class AddCustomerCommand extends Command {
 
-    public static final String COMMAND_WORD = "add customer";
+    public static final String COMMAND_WORD = "addCustomer";
```
Comment (bbryant824) [inline src/main/java/seedu/address/logic/commands/AddCustomerCommand.java:17]: appropriate use of COMMAND_WoRD

File: src/main/java/seedu/address/logic/parser/AddCustomerCommandParser.java [hunk 2]
```diff
@@ -40,14 +40,8 @@ public AddCustomerCommand parse(String args) throws ParseException {
...
-        // Add print statements to log the parsed values
-        System.out.println("Parsed name: " + name);
-        System.out.println("Parsed phone: " + phone);
-        System.out.println("Parsed email: " + email);
-        System.out.println("Parsed address: " + address);
-        System.out.println("Parsed preference: " + preference);
-        System.out.println("Parsed information: " + information);
-        System.out.println("Parsed tags: " + tagList);
+        // Add the "customer" tag explicitly
+        tagList.add(new Tag("customer"));
```
Comment (bbryant824) [inline src/main/java/seedu/address/logic/parser/AddCustomerCommandParser.java:44]: Well done! Keep it up

File: src/main/java/seedu/address/model/person/Customer.java [hunk 3]
```diff
@@ -66,10 +67,10 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder(super.toString());
-        builder.append(" Information: ")
-                .append(getInformation());
+        builder.append(" Information: ").append(getInformation());
```
Comment (bbryant824) [inline src/main/java/seedu/address/model/person/Customer.java:70]: Good use of StringBuilder for commands.

File: src/main/java/seedu/address/ui/PersonCard.java [hunk 4]
```diff
@@ -58,8 +50,21 @@ public PersonCard(Person person, int displayedIndex) {
         preference.setText(person.getPreference().value);
         email.setText(person.getEmail().value);
         remark.setText(person.getRemark().value);
+
+        // Sort and display the tags
```
Comment (bbryant824) [inline src/main/java/seedu/address/ui/PersonCard.java:54]: Well coded on this part.

**Review by bbryant824 — APPROVED** [review]
Good effort Lucas! Keep it up.

File: src/main/java/seedu/address/logic/commands/AddCustomerCommand.java
Comment (ryenl) [inline src/main/java/seedu/address/logic/commands/AddCustomerCommand.java:1]: Nice change to the command format to match the other commands

File: src/main/java/seedu/address/logic/commands/AddSupplierCommand.java
Comment (ryenl) [inline src/main/java/seedu/address/logic/commands/AddSupplierCommand.java:1]: Same as above. Keep it up!

File: src/main/java/seedu/address/logic/parser/AddSupplierCommandParser.java
Comment (ryenl) [inline src/main/java/seedu/address/logic/parser/AddSupplierCommandParser.java:1]: Nice use of tagging to reflect the supplier

**Review by ryenl — APPROVED** [review]

**Review by ryenl — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/person/Customer.java [hunk 5]
```diff
@@ -66,10 +67,10 @@ public int hashCode() {
     @Override
     public String toString() {
         final StringBuilder builder = new StringBuilder(super.toString());
```
Comment (samuelthen) [inline src/main/java/seedu/address/model/person/Customer.java:69]: Utilization of StringBuilding for better efficiency and neatness

File: src/main/resources/view/DarkTheme.css [hunk 6]
```diff
@@ -342,6 +342,21 @@
     -fx-vgap: 3;
 }
 
+.tag-customer {
+    -fx-background-color: red !important;
```
Comment (samuelthen) [inline src/main/resources/view/DarkTheme.css:346]: Perhaps can use a more modern color?

File: src/main/resources/view/DarkTheme.css [hunk 7]
```diff
@@ -342,6 +342,21 @@
...
+    -fx-text-fill: white !important;
+    -fx-padding: 2px 5px !important;
+    -fx-border-radius: 4px !important;
+    -fx-background-radius: 4px !important;
+}
+.tag-supplier {
+    -fx-background-color: green !important;
```
Comment (samuelthen) [inline src/main/resources/view/DarkTheme.css:353]: Similar to previous comments, using pure green might be too bright

**Pull request merged**